improver_tests/ensemble_copula_coupling/test_ResamplePercentiles.py

```diff
@@ -431,20 +429,26 @@ def test_check_data_spot_forecasts(self):
         data values for the percentiles for spot forecasts.
         """
         spot_percentile_cube = set_up_spot_test_cube(cube_type="percentile")
-        spot_percentile_cube.data = np.tile(np.linspace(5, 10, 3), 9).reshape(3, 9)
+        spot_percentile_cube.data = (
+            np.tile(np.linspace(5, 10, 3), 9).reshape(3, 9) + 273.15
+        )
 
-        data = np.array(
-            [
-                [5.0, 7.5, 10.0, 5.0, 7.5, 10.0, 5.0, 7.5, 10.0],
-                [5.0, 7.5, 10.0, 5.0, 7.5, 10.0, 5.0, 7.5, 10.0],
-                [5.0, 7.5, 10.0, 5.0, 7.5, 10.0, 5.0, 7.5, 10.0],
-            ]
+        data = (
+            np.array(
+                [
+                    [5.0, 7.5, 10.0, 5.0, 7.5, 10.0, 5.0, 7.5, 10.0],
+                    [5.0, 7.5, 10.0, 5.0, 7.5, 10.0, 5.0, 7.5, 10.0],
+                    [5.0, 7.5, 10.0, 5.0, 7.5, 10.0, 5.0, 7.5, 10.0],
+                ]
+            )
+            + 273.15
         )
         percentiles = spot_percentile_cube.coord("percentile").points
+
         result = Plugin()._interpolate_percentiles(
-            spot_percentile_cube, percentiles, self.bounds_pairing, self.perc_coord
+            spot_percentile_cube, percentiles, self.perc_coord
         )
-        self.assertArrayAlmostEqual(result.data, data)
+        self.assertArrayAlmostEqual(result.data, data, decimal=5)
```

Comment:
The default is 6, which was failing because 32-bit float machine precision is 7 significant figures, which is just on the limit of what this test was accepting.